src/sph_geom/mod.rs

```diff
@@ -542,14 +512,14 @@ mod tests {
   fn test_intersect_parallel() {
```

Comment:
This test passes but I am not sure, is it enough tested the special_points_finder::intersect_small_circle method ?

Reply:
An idea of the best implementation between `special_point_finder::intersect_small_circle` and `sph_geom::intersect_parallel`?
Both seems to have advantages compared to the other one (flat approximation in `intersect_small_circle`, all cases handled in `intersect_parallel`)...

Reply:
I think the special_point_finder is enough for me yes